### ISSUE TITLE
Bug - Exiting the shell gets stuck and causes an exception in 0.1.23.

### ISSUE THREAD

**Issue by ghost** [romgrk/termrk#50]
Hello

Entering `exit` to quit the current shell correctly prints `Process terminated` but then it seems to get stuck. Pressing the return key then generates the following error:

```
Uncaught TypeError: Cannot read property 'send' of null
At ~/.atom/packages/Termrk/lib/termrk-model.coffee:92

TypeError: Cannot read property 'send' of null
  at TermrkModel.module.exports.TermrkModel.write (~/.atom/packages/Termrk/lib/termrk-model.coffee:92:13)
  at Terminal.<anonymous> (~/.atom/packages/Termrk/lib/termrk-view.coffee:121:45)
  at Terminal.EventEmitter.emit (~/.atom/packages/Termrk/node_modules/term.js/src/term.js:109:12)
  at Terminal.handler (~/.atom/packages/Termrk/node_modules/term.js/src/term.js:2898:8)
  at Terminal.keyDown (~/.atom/packages/Termrk/node_modules/term.js/src/term.js:2629:8)
  at TermrkView.module.exports.TermrkView.inputKeydown (~/.atom/packages/Termrk/lib/termrk-view.coffee:147:37)
  at TermrkView.inputKeydown (~/.atom/packages/Termrk/lib/termrk-view.coffee:2:1)
```

Thanks in advance

**Comment by romgrk** [comment]
Damn 0.1.23.

**Comment by romgrk** [comment]
Patched by 124d93b603628266f8cfa74dc7a7be33b8be6006.
